vision/automl/src/main/java/com/google/cloud/vision/samples/automl/PredictionApi.java

```diff
@@ -65,76 +61,76 @@
+      Image image = Image.newBuilder().setImageBytes(content).build();
+      ExamplePayload examplePayload = ExamplePayload.newBuilder().setImage(image).build();
 
-    // Additional parameters that can be provided for prediction e.g. Score Threshold
-    Map<String, String> params = new HashMap<>();
-    if (scoreThreshold != null) {
-      params.put("score_threshold", scoreThreshold);
-    }
-    // Perform the AutoML Prediction request
-    PredictResponse response = predictionClient.predict(name, examplePayload, params);
-
-    System.out.println("Prediction results:");
-    for (AnnotationPayload annotationPayload : response.getPayloadList()) {
-      System.out.println("Predicted class name :" + annotationPayload.getDisplayName());
-      System.out.println(
-          "Predicted class score :" + annotationPayload.getClassification().getScore());
+      // Additional parameters that can be provided for prediction e.g. Score Threshold
+      Map<String, String> params = new HashMap<>();
+      if (scoreThreshold != null) {
+        params.put("score_threshold", scoreThreshold);
+      }
+      // Perform the AutoML Prediction request
+      PredictResponse response = predictionClient.predict(name, examplePayload, params);
+
+      System.out.println("Prediction results:");
+      for (AnnotationPayload annotationPayload : response.getPayloadList()) {
+        System.out.println("Predicted class name :" + annotationPayload.getDisplayName());
+        System.out.println(
+            "Predicted class score :" + annotationPayload.getClassification().getScore());
+      }
+    } catch (IOException e) {
+      e.printStackTrace();
     }
   }
   // [END automl_vision_predict]
 
-  public static void main(String[] args) throws IOException {
-    PredictionApi predictionApi = new PredictionApi();
-    predictionApi.argsHelper(args, System.out);
+  public static void main(String[] args) {
+    argsHelper(args);
   }
 
-  public static void argsHelper(String[] args, PrintStream out) throws IOException {
+  static void argsHelper(String[] args) {
     ArgumentParser parser =
         ArgumentParsers.newFor("PredictionApi")
             .build()
             .defaultHelp(true)
             .description("Prediction API Operation");
-    Subparsers subparsers = parser.addSubparsers().dest("command");
 
-    Subparser predictParser = subparsers.addParser("predict");
-    predictParser.addArgument("modelId");
-    predictParser.addArgument("filePath");
-    predictParser.addArgument("scoreThreshold").nargs("?").type(String.class).setDefault("");
+    parser.addArgument("modelId").required(true);
+    parser.addArgument("filePath").required(true);
+    parser.addArgument("scoreThreshold").nargs("?").type(String.class).setDefault("");
 
-    String projectId = System.getenv("PROJECT_ID");
+    String projectId = System.getenv("GOOGLE_CLOUD_PROJECT");
     String computeRegion = System.getenv("REGION_NAME");
```

Comment:
I'm fine w/ this, but don't we have a standard of ZONE & REGION, if not, should we?

Reply:
Sorry, clarification question: do you mean like the var name should be `REGION` or like we should provide a suggested default region: `us-central1`?